lib/sandboxed_renderer/init.js

```diff
@@ -103,4 +106,6 @@ if (preloadSrc) {
   const preloadFn = geval(preloadWrapperSrc)
   const {setImmediate, clearImmediate} = require('timers')
   preloadFn(preloadRequire, preloadProcess, Buffer, global, setImmediate, clearImmediate)
+} else if (preloadError) {
+  console.error(preloadError.stack)
```

Comment:
@miniak is this how you wanted to handle preload error? I can't think of a better way right now, logging via `console.error` seems like a sane way to do it, since it would be available in the terminal if electron is started with `--enable-logging`